src/video/SDL_sysvideo.h

```diff
@@ -107,12 +107,13 @@ struct SDL_Window
     int last_pixel_w, last_pixel_h;
     Uint32 flags;
     SDL_bool fullscreen_exclusive;  /* The window is currently fullscreen exclusive */
-    SDL_bool last_fullscreen_exclusive;  /* The last fullscreen_exclusive setting */
+    SDL_DisplayID last_fullscreen_exclusive;  /* The last fullscreen_exclusive display */
     SDL_DisplayID last_displayID;
 
     /* Stored position and size for windowed mode */
     SDL_Rect windowed;
 
+    SDL_DisplayMode desired_mode;
```

Comment:
Suggest renaming to `requested_fullscreen_mode`

Reply:
or `desired_fullscreen_mode` and then renaming the other variable `current_fullscreen_mode`